build-go.sh

```diff
@@ -315,15 +315,26 @@ function startGoBuilds() {
 	versions="$4"
 	existing="$5"
 
+	if [ ! -e "$versions" ]; then
+		fail "versions file $versions does not exist"
+	fi
+
+	if ! mkdir -p "$releases"; then
+		fail "failed to create releases directory"
+	fi
+
 	if [ -z "$existing" ]; then
 		existing="/ipns/dist.ipfs.io"
 	fi
-	echo "using $existing as a template."
 
+	echo "comparing $versions with $existing/$distname/versions"
+	newVersions=$(comm -13 <(ipfs cat "$existing/$distname/versions") "$versions")
```

Comment:
I should add a check to see if `ipfs` exists and the deamon is online.

Reply:
Added a check to deps-check.sh and updated it so we check it early and halt the build if any are missing in https://github.com/ipfs/distributions/pull/320/commits/3398a234ce493f8fa780a3873e2d5b222e1bbfee